boards/arm/b_l4s5i_iot01a/pinmux.c

```diff
@@ -18,6 +18,39 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PB6, STM32L4X_PINMUX_FUNC_PB6_USART1_TX},
 	{STM32_PIN_PB7, STM32L4X_PINMUX_FUNC_PB7_USART1_RX},
 #endif
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(uart4), okay) && CONFIG_SERIAL
```

Comment:
Please update doc to reflect this.

Reply:
UART 4 added to the index.rst  "Default Zephyr Peripheral Mapping"